src/validation/ValidationExecutor.ts

```diff
@@ -184,6 +184,11 @@ export class ValidationExecutor {
     }
     
     private nestedValidations(value: any, metadatas: ValidationMetadata[], errors: ValidationError[]) {
+
+        if (value === void 0) {
```

Comment:
It's mostly a browser based issue. For old browsers that do not support JS 1.8.5 you could do something like:
```js
console.log(undefined);
var undefined = 1;
console.log(undefined); // Would output 1
```
Read [this](http://stackoverflow.com/a/19369078/1378525) for details